src/scripts/main.js

```diff
@@ -1,3 +1,42 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const body = document.querySelector('body');
+
+const promise = new Promise((resolve, reject) => {
+  resolve(myDisplayer);
+
+  reject(myDisplayer2);
+});
+
+promise.then(
+  function (value) {
+    myDisplayer(value);
+  },
+  function (error) {
+    myDisplayer2(error);
+  },
```

Comment:
The third parameter in the `then` method is not valid. The `then` method only accepts two arguments: a callback for when the promise is resolved and a callback for when the promise is rejected. Remove the third parameter.